library/include/hipblaslt-ext.hpp

```diff
@@ -117,6 +120,56 @@ namespace hipblaslt_ext
             = 0; //!< The aux batch stride. Only works if mode is set to aux related epilogues.
     };
 
+    /*! \ingroup types_module
+     *  \brief hipblasLt extension Epilogue for gemm problems.
+     *
+     * \details This class sets the epilogue of a gemm problem.
+     */
+    class GemmEpilogueV2
+    {
+    public:
+        HIPBLASLT_EXPORT GemmEpilogueV2();
+        HIPBLASLT_EXPORT ~GemmEpilogueV2();
+
+        HIPBLASLT_EXPORT                 GemmEpilogueV2(const GemmEpilogueV2& epilogue);
+        HIPBLASLT_EXPORT GemmEpilogueV2& operator=(const GemmEpilogueV2& epilogue);
+
+        HIPBLASLT_EXPORT                 GemmEpilogueV2(GemmEpilogueV2&& epilogue);
+        HIPBLASLT_EXPORT GemmEpilogueV2& operator=(GemmEpilogueV2&& epilogue);
+
+        HIPBLASLT_EXPORT void
+            setMode(hipblasLtEpilogue_t mode); //!< Set the mode of epilogue. Default is gemm.
+        HIPBLASLT_EXPORT void setBiasDataType(
+            hipDataType
+                biasDataType); //!< Set the bias datatype. Only works if mode is set to bias related epilogues.
+        HIPBLASLT_EXPORT void setAuxLeadingDimension(
+            int auxLeadingDimension); //!< Set the aux leading dimension. Only works if mode is set to aux related epilogues.
+        HIPBLASLT_EXPORT void setAuxBatchStride(
+            int auxBatchStride); //!< Set the aux batch stride. Only works if mode is set to aux related epilogues.
+        HIPBLASLT_EXPORT void setScalingAType(
+            int scalingAType); //!< 0 is scalar, 1 is vector. Only works if DataTypeA = DataTypeB = FP8.
+        HIPBLASLT_EXPORT void setScalingBType(
+            int scalingBType); //!< 0 is scalar, 1 is vector. Only works if DataTypeA = DataTypeB = FP8.
+
+        HIPBLASLT_EXPORT hipblasLtEpilogue_t
```

Comment:
also needs return status for Get API and return actual user requested data in the pointer

Reply:
Too C style, the private members are just structures,

Reply:
OK